package.json

```diff
@@ -6,24 +6,18 @@
   "scripts": {
     "dev": "yarn && nodemon --watch 'src/**/*.ts' --exec 'ENV=dev ts-node -r tsconfig-paths/register' src/server.ts",
     "prebuild": "rm -rf /dist",
-    "build": "tsc && cp -R src/admin dist/src && cp ./ormconfig.js dist/",
+    "build": "tsc && cp -R src/admin dist/src",
     "start": "ENV=dev node -r tsconfig-paths/register -r ts-node/register ./dist/src/server.js",
     "prod": "ENV=prod node -r ./prod-paths.js ./dist/src/server.js",
     "typeorm": "ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js",
     "test": "ENV=test jest --runInBand --useStderr",
     "test:cover": "yarn test --coverage",
     "test:ci": "ENV=ci jest --coverage",
-    "db:setup": "yarn schema:sync && yarn seed:run",
-    "db:reset": "yarn schema:drop && yarn db:setup",
+    "db:setup": "yarn typeorm schema:sync && yarn seed:run",
+    "db:reset": "yarn typeorm schema:drop && yarn db:setup",
     "lint": "eslint \"src/**\" --color",
     "seed:config": "ts-node ./node_modules/typeorm-seeding/dist/cli.js config",
-    "seed:run": "ts-node -r tsconfig-paths/register ./node_modules/typeorm-seeding/dist/cli.js seed",
-    "schema:drop": "ts-node ./node_modules/typeorm/cli.js schema:drop",
-    "schema:sync": "ts-node ./node_modules/typeorm/cli.js schema:sync",
-    "migration:generate": "yarn typeorm migration:create -n",
-    "migration:revert": "yarn typeorm migration:revert",
-    "migration:show": "yarn typeorm migration:show",
-    "migration:run": "yarn typeorm migration:run"
+    "seed:run": "ts-node -r tsconfig-paths/register ./node_modules/typeorm-seeding/dist/cli.js seed"
```

Comment:
Maybe is a good idea a `post` typeorm command in order to format the migrations with prettier (currently the migrations are in one line)
```
"posttypeorm": ""prettier -w src/database/migrations"
```

Reply:
WDYT?

Reply:
Done!